### PR TITLE
Init fixtures data

### PR DESCRIPTION
**Pull request by vinyll**
Right now this is implementing fixture creation via a task.
Next step will be to fix `udata init` and prompt for fixtures.

### CODE REVIEW

File: udata/commands/fixtures.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        DatasetDiscussionFactory(subject=dataset)
+
+
+def generate_reuses(count, user=None):
+    for _ in range(0, count):
+        VisibleReuseFactory(owner=user)
+
+
+def generate_licenses(count):
+    for _ in range(0, count):
```
Comment (davidbgk) [inline udata/commands/fixtures.py]: Why not `.create_batch(count)`?

**Comment by davidbgk** [comment]
With a fresh update on `dev`, I have this error when I serve the project:

```ERROR: Unable to import init: No module named fixtures```

Not sure if it's related though.

**Comment by vinyll** [comment]
Let me check that a file is not missing

File: udata/commands/fixtures.py (hunk 1 above)
Comment (vinyll) [inline udata/commands/fixtures.py]: Because it does not exist for factories. That's a model's method.

**Comment by noirbizarre** [comment]
Ready to merge ?

**Comment by vinyll** [comment]
r+?

**Comment by noirbizarre** [comment]
r+ :+1:

**Pull request merged**